perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -129,15 +134,20 @@ def _parse_results(folders, results_list, message, num_samples) -> dict:
 
   Args:
     folders: List containing protobufs of testing folders.
-    results_list: Dictionary containing the list of results (for all samples)
-                  for each testing folder.
+    result_list: Dictionary containing the list of start and end times
+                 (for all samples) for each testing folder.
     message: String which describes/titles the test.
     num_samples: Number of samples to collect for each test.
 
   Returns:
     A dictionary containing the various metrics in a JSON format.
   """
 
+  # Dictionary containing the latencies (for all samples) for each testing folder
+  results_list = {}
+  for testing_folder in folders:
+    results_list[testing_folder.name] = [(row[1] - row[0])*1000 for row in result_list[testing_folder.name]]
```

Comment:
Try to avoid numeric numbers in the code, instead create a const variable for 1000 and use it here.

Reply:
Also, row[1] or row[0] is not readable. Either create (or check if already exist) a struct with field (start_time or end_time) or just write one liner, what does row[0], row[1] signify?